### PR TITLE
[TASK] Update rippled.cfg for rippled v0.28.0

### PR DESCRIPTION
**Pull request by wilsonianb**

### CODE REVIEW

**Comment by wilsonianb** [comment]
+r

File: rippled.cfg [hunk 1]
```diff
@@ -3,31 +3,25 @@
...
 ssl_key = /etc/ssl/private/server.key
 ssl_chain = /etc/ssl/certs/server.crt
 
 [port_rpc]
 port = 51234
 ip = 0.0.0.0
-admin = allow
-protocol = http,https
+admin = 127.0.0.1
+protocol = https
```
Comment (mtrippled) [inline rippled.cfg]: Should allow http still at least for testing.

Comment (wilsonianb) [inline rippled.cfg]: fixed

**Comment by wilsonianb** [comment]
+r

**Pull request merged**